test/format/verify.c

```diff
@@ -47,7 +47,8 @@ table_verify(TABLE *table, void *arg)
     wt_wrap_open_session(conn, &sap, table->track_prefix,
       enable_session_prefetch() ? SESSION_PREFETCH_CFG_ON : NULL, &session);
     ret = session->verify(session, table->uri, "strict");
-    testutil_assert(ret == 0 || ret == EBUSY);
+    if (!g.disagg_storage_config)
```

Comment:
@mariammojid I’m aware that verify doesn’t work in follower mode. However, I’d like to run mirrored tables on the follower, which requires verify to be enabled. Is there an existing ticket for adding verify support in follower mode that I can link here as FIXME? Once verify is supported for follower ingest tables, we can remove this.

FYI, This is just a draft, I’ll add a better conditional later.

Reply:
Ok so the error I am getting is `ENOENT`. I believe a follower can have the stable table empty for sometime? Until leader has done a checkpoint for the follower to read.

Reply:
thanks @mariammojid please let me know the ticket number so that I could be unblocked here. During that ticket we can resolve the logic here thanks.

Reply:
[WT-15413](https://jira.mongodb.org/browse/WT-15413) Verify accounts for followers missing stable constituent prior to checkpoint pickup